docs/quick/k8s/submarine-kubeflow.md

```diff
@@ -0,0 +1,112 @@
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Submarine with Kubeflow Tutorial
+Submarine with Kubeflow runs on **Linux** and **macOS**. To be able to run Submarine, the requirements are:
+* Java 8.x
+* Kubeflow 0.6+
+* Submarine 0.3.0
+
+## Prepare K8s cluster
+We recommend to use [`kind`](https://kind.sigs.k8s.io/) to create the K8s cluster using Docker contianer "node".
+```
+kind create cluster --image kindest/node:v1.15.6 --name submarine
+```
+
+> NOTICE: Kuebflow 0.7 compatible with kubernetes 1.14 and 1.15
+
+### Deploy Dashboard
+To deploy Dashboard, execute following command:
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta8/aio/deploy/recommended.yaml
+```
+
+### Create RBAC
```

Comment:
Is this step belongs to "Deploy Dashboard"? If so, it should be "#### Create RBAC". Four "#". And please change following titles if needed.